package-lock.json

```diff
@@ -45,12 +45,14 @@
       }
     },
     "ajv": {
-      "version": "4.11.8",
-      "resolved": "https://registry.npmjs.org/ajv/-/ajv-4.11.8.tgz",
-      "integrity": "sha1-gv+wKynmYq5TvcIK8VlHcGc5xTY=",
+      "version": "5.2.2",
```

Comment:
Yes. But running `npm install metasync metarhia-common` will add them to `package.json` too.